packages/@okta/vuepress-site/docs/guides/sign-into-web-app/main/index.md

```diff
@@ -24,90 +24,48 @@ This guide explains how to use Okta as the user store for your server-side web a
 * Knowledge of building server-side Web applications
 * A project or application to add authentication to.
 
-**Sample code**
-
-<StackSelector snippet="samplecode" noSelector/>
-
 ---
 
-## Refresh tokens and web apps
-
-With browser-based apps, the risk of the refresh token being compromised is high when a persistent refresh token is used. This threat is greatly reduced by rotating refresh tokens. [Refresh token rotation](/docs/guides/refresh-tokens/main/#refresh-token-rotation) helps a public client to securely rotate refresh tokens after each use. A new refresh token is returned each time the client makes a request to exchange a refresh token for a new access token. Refresh token rotation works with SPAs, native apps, and web apps in Okta.
-
-See the [OAuth 2.0 for Browser-Based Apps specification](https://tools.ietf.org/html/draft-ietf-oauth-browser-based-apps-05#page-10) for the latest spec information on using refresh tokens with browser-based apps.
-
-## Understand the callback route
```

Comment:
This isn't really needed with Spring Boot since Spring Security defines the callback for you. You just need to set it as a redirect URI.